src/components/TextInput/TextInput.tsx

```diff
@@ -199,7 +199,7 @@ export const TextInput = ({
             {...focusProps}
             {...props}
             className={merge([
-                'tw-flex tw-items-center tw-h-9 tw-gap-2 tw-px-3 tw-border tw-transition tw-rounded tw-text-s tw-font-sans tw-relative tw-bg-white dark:tw-bg-transparent',
+                'tw-flex tw-items-center tw-h-9 tw-gap-2 tw-px-3 tw-border tw-transition tw-rounded tw-text-sm tw-font-sans tw-relative tw-bg-white dark:tw-bg-transparent',
```

Comment:
Indeed the class is not valid but you're replacing it with a Tailwind default. Could you try using our typography alias tokens -> `tw-text-body-small` instead.